sympy/physics/hydrogen.py

```diff
@@ -132,11 +132,11 @@ def Psi_nlm(n, l, m, r, phi, theta, Z=1):
     # sympify arguments
     n, l, m, r, phi, theta, Z = map(S, [n, l, m, r, phi, theta, Z])
     # check if values for n,l,m make physically sense
-    if n.is_integer and n < 1:
+    if n.is_integer == False or (n < 1) == True:
```

Comment:
May be, `is False` and `is True` would be faster as it does the comparison by reference.

Reply:
In other places too in the diff.

Reply:
> May be, `is False` and `is True` would be faster as it does the comparison by reference.

I tried that but I think the tests failed in that case on running either `flake8 `or `bin/quality`.